frontend/src/components/VFourOhFour.vue

```diff
@@ -1,15 +1,10 @@
 <template>
   <div class="error grid overflow-x-hidden">
-    <svg
+    <VSvg
       class="z-0 pointer-events-none col-start-1 row-start-1 -mx-[15%] fill-dark-charcoal opacity-5 lg:mx-15 lg:-mt-20"
-      xmlns="http://www.w3.org/2000/svg"
       viewBox="0 0 1320 569"
-      aria-hidden="true"
-      focusable="false"
-    >
-      <use :href="`${Oops}#oops`" />
```

Comment:
Should this href have been deleted?

Reply:
Yes, the `<VSvg>` component handles finding the `href`s now, and we do not use the id of the inner SVG path. We only need to pass the name of the file to use it as an SVG.